### ISSUE TITLE
False advertisement

### ISSUE THREAD

**Issue by ghost** [vulkanprogrammingguide/examples#2]
This book was clearly marketed with misleading claims that the code was already available. Six months later there is nothing.

**Comment by jordisimon** [comment]
I agree. 
From his webpage (http://www.vulkanprogrammingguide.com/): "Example code can be found here: https://github.com/vulkanprogrammingguide/examples"

I feel scammed...

**Comment by grim210** [comment]
I mean...is there any code at all?  Incomplete code would be better than no code.

**Comment by tiantian20007** [comment]
Couldn't agree any more.

**Comment by JanPlusPlus** [comment]
I also agree. It's now over 8 months ago and there is no code yet. An incomplete code would be allright...
@grahamsellers please update the status of this project that we know if or when the code releases

**Comment by AZimmermann89** [comment]
9 Months. Still no code.

**Comment by Siri94** [comment]
Here is two alternative repositories:
https://github.com/LunarG/VulkanSamples
https://github.com/SaschaWillems/Vulkan
Good Luck.

**Comment by mmostajab** [comment]
No codes... no wisdom... Just you, fratello mio (my brother). Requiescat in pace, Vulkan.

**Comment by Ident** [comment]
The fact that this book STILL has no accompanying samples and that it mostly just replicates the API documentation without too much explanation on how to actually put everything together to form a meaningful piece of code to render or compute something, lead me to buy another book: Vulkan Cookbook

I haven't read too much into it yet but I find it more extensive and a better addition to the official Vulkan API docu, than what I saw in samples of this Vulkan Programming book.

And guess what, that other author even managed to put up samples:
https://github.com/PacktPublishing/Vulkan-Cookbook

It is a shame that the official book's authors did not manage that -they could have at least paid someone to make the samples, overseeing the development of those. Can't be too difficult to do that can it?

Well, we will see. If they make a second edition and put up good examples and add more info to the book I might buy it one day. Until then: SHAME!

**Comment by dstewart** [comment]
I agree with the above post. I was able to produce working code using this book, but only after 6 months of totally uninterupted programming. It was not a good use of my time, to put it mildly...

**Comment by mmostajab** [comment]
@dstewart It would be very helpful if you share whatever you have written to reproduce working code. We also can contribute to improve it.

**Comment by dstewart** [comment]
@mmostajab see https://github.com/dstewart/Vulture

Be warned that the code is not unit tested

**Comment by GarethH96** [comment]
@dstewart Link gives me 404 error.

**Comment by dstewart** [comment]
@GarethH96 Sorry, I cancelled this project a couple months ago. My original intent was to produce a commercial game engine but it proved to be too much work for one developer. Perhaps with some extra assistance I can revisit the topic this year.
